src/main/drivers/bus_quadspi.c

```diff
@@ -155,6 +155,61 @@ const quadSpiHardware_t quadSpiHardware[] = {
         .rcc = RCC_AHB3(QSPI),
     },
 #endif
+
+#ifdef STM32F745
```

Comment:
You can not use target specific defines (STM32F745) anywhere in the code. Use the proper MCU types instead: `STM32F722xx` / `STM32F745xx` / `STM32F746xx` / `STM32F765xx`

Is there a reason that this can not work on STM32F722?